test/mocha/25-http-edv-client.js

```diff
@@ -242,6 +242,82 @@ describe('bedrock-edv-storage HTTP API - edv-client', () => {
     });
   }); // end `update`
 
+  describe('update config', () => {
+    it('should update an EDV config', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f678';
+      const handle = 'testKey3';
+      const {httpsAgent} = brHttpsAgent;
+
+      const capabilityAgent = await CapabilityAgent.fromSecret(
+        {secret, handle});
+
+      const keystoreAgent = await helpers.createKeystore({capabilityAgent});
+      const invocationSigner = capabilityAgent.getSigner();
+
+      // corresponds to the passport authenticated user
+      const actor = actors['alpha@example.com'];
+
+      let edvClient;
+      let edvConfig;
+      let config;
+      let err;
+      try {
+        ({edvClient, edvConfig} = await helpers.createEdv(
+          {actor, capabilityAgent, keystoreAgent, urls}));
+        config = await EdvClient.findConfigs({
+          controller: edvConfig.controller, invocationSigner,
+          url: edvClient.id, httpsAgent
+        });
+        config.sequence++;
+        await EdvClient.updateConfig({
+          id: config.id, config, httpsAgent
+        });
+      } catch(e) {
+        err = e;
+      }
+      // no response is returned from sucessful update
+      assertNoError(err);
+    });
+    it('should not update an EDV config with wrong id', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f678';
+      const handle = 'testKey3';
+      const {httpsAgent} = brHttpsAgent;
+
+      const capabilityAgent = await CapabilityAgent.fromSecret(
+        {secret, handle});
+
+      const keystoreAgent = await helpers.createKeystore({capabilityAgent});
+      const invocationSigner = capabilityAgent.getSigner();
+
+      // corresponds to the passport authenticated user
+      const actor = actors['alpha@example.com'];
+
+      let edvClient;
+      let edvConfig;
+      let config;
+      let err;
+      try {
+        ({edvClient, edvConfig} = await helpers.createEdv(
+          {actor, capabilityAgent, keystoreAgent, urls}));
+        config = await EdvClient.findConfigs({
+          controller: edvConfig.controller, invocationSigner,
+          url: edvClient.id, httpsAgent
+        });
+        const url = config.id;
+        config.id = '123';
+        config.sequence += 1;
```

Comment:
Same change as above
```suggestion
        config.sequence++;
```

Reply:
Updated.